includes/Core/Assets/Assets.php

```diff
@@ -401,12 +376,7 @@ private function get_assets() {
 					'dependencies' => array( 'wp-i18n' ),
 					'execution'    => 'defer',
 					'before_print' => function( $handle ) {
-						$inline_data = $this->get_inline_base_data();
-						wp_add_inline_script(
-							$handle,
-							'window._googlesitekitBase = ' . wp_json_encode( $inline_data ),
-							'before'
-						);
+						wp_localize_script( $handle, '_googlesitekitBase', $this->get_inline_base_data() );
```

Comment:
Could we not leave this to use `wp_add_inline_script()` instead? This one still has a `src`, and I'd prefer to use the more proper function wherever possible.